### DISCUSSION TITLE
Expecting value Error

### DISCUSSION THREAD

**Discussion by MxPenguin464** [nianeyna/ao3downloader#145] [Q&A] [answered]
![AO3Help](https://github.com/nianeyna/ao3downloader/assets/172455454/027a7790-ef7e-4fc3-bc25-3dcf3db0b486)
I tried to start a new download session, but after it comes to logging in, I get the following string "Expecting value: line 1 column 1 (char 0)" and then it goes back to the "beginning. Is this a bug or have I messed something up? I have tried both logging in and not logging in.

**Comment by nianeyna** [comment]
❗ Checkpoint! ❗

Have you done ALL of the troubleshooting steps in the README? (The README is the explanatory text on the landing page for ao3downloader. You can find the Troubleshooting section using the Table of Contents at the beginning of the README.)

If you have done all the troubleshooting steps, please tell me what the results were. If you haven't, please do them to the best of your ability, and tell me what the results were. ✨ 

❗ Thank you! ❗

**Reply by MxPenguin464** [reply]
The error message in the log reads:
{"error": "Expecting value: line 1 column 1 (char 0)", "message": "Problem parsing log file during initial setup", "stacktrace": "Traceback (most recent call last):\n  File 

followed by a very long string of my file location.

**Reply by nianeyna** [reply]
Can you delete the "logs" folder and try again?

**Reply by MxPenguin464** [reply]
![AO3Help2](https://github.com/nianeyna/ao3downloader/assets/172455454/56cd26e6-b1fc-43ae-9dc1-705467f4f012)
I think I made it worse. I tried to delete the log, and didn't have a different result, so tried deleting venv, no luck. I tried deleting the file completely and redownloading and got this.

**Reply by nianeyna** [reply]
Huh ok to clarify - the second screenshot is from a completely new version of the code that you downloaded from github?

**Reply by MxPenguin464** [reply]
Yes, this is the new version.

**Reply by nianeyna** [reply]
well, darn. I'm not sure what the problem is if you've done all the troubleshooting steps. nothing additional is jumping out to me - on the second try it's clear that your virtual environment did not initialize correctly, but I'm not sure why not, or why it worked (sort of) the first time and not this time. I'll ponder this and let you know if I come up with any more ideas that you can try.

**Reply by MxPenguin464** [reply]
Ok, thanks!

**Reply by MxPenguin464** [reply, marked as answer]
> well, darn. I'm not sure what the problem is if you've done all the troubleshooting steps. nothing additional is jumping out to me - on the second try it's clear that your virtual environment did not initialize correctly, but I'm not sure why not, or why it worked (sort of) the first time and not this time. I'll ponder this and let you know if I come up with any more ideas that you can try.

I just wanted to let you know that I did get it working. Turns out I was just being impatient when I ran the cmd on the re-install, which I'm assuming led to the virtual environment problem. Either way, it works now!

**Reply by nianeyna** [reply]
Yay! thank you so much for letting me know you got it, I do worry about these things. And thank you for the coffee as well!
